From 1852483756bf9faeb24133b2bfb107602bf35cda Mon Sep 17 00:00:00 2001
From: Tobias <ruck.tobias@gmail.com>
Date: Tue, 13 Apr 2021 09:36:59 -0500
Subject: [PATCH 078/196] Enforce Logos coinbase rule on testnet (#143)

Testnet coins are worthless anyway; this will allow mining pools to test their setup.
It'll also allow us to test new coinbase outputs on the testnet.
---
 src/chainparams.cpp | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/chainparams.cpp b/src/chainparams.cpp
index 880553574..65a66073a 100644
--- a/src/chainparams.cpp
+++ b/src/chainparams.cpp
@@ -238,8 +238,8 @@ public:
             .nTimeout = 1230767999,
         };
 
-        // The miner fund is disabled by default on testnet.
-        consensus.enableMinerFund = false;
+        // The miner fund is enabled by default on testnet.
+        consensus.enableMinerFund = ENABLE_MINER_FUND;
 
         // The best chain should have at least this much work.
         consensus.nMinimumChainWork =
-- 
2.39.5 (Apple Git-154)

